tests/js/session-tests.js

```diff
@@ -1033,6 +1039,116 @@ module.exports = {
         });
     },
 
+    async testClientResetAtOpen() {
+        // FIXME: try to enable for React Native
+        if (!isNodeProcess) {
+            return;
+        }
+
+        if (!global.testAdminUserInfo) {
+            throw new Error("Test requires an admin user");
+        }
+
+        let called = false;
+        const credentials = Realm.Sync.Credentials.usernamePassword(global.testAdminUserInfo.username, global.testAdminUserInfo.password);
+        let user = await Realm.Sync.User.login('http://127.0.0.1:9080', credentials);
+        var realm;
+        const config = user.createConfiguration({ sync: { url: 'realm://127.0.0.1:9080/~/myrealm' } });
+        config.schema = [schemas.IntOnly];
+        config.sync.clientResyncMode = 'manual';
+        config.sync.fullSynchronization = true;
+        config.sync.error = (sender, error) => {
+            called = true;
+            try {
+                TestCase.assertEqual(error.name, 'ClientReset');
+                TestCase.assertDefined(error.config);
+                TestCase.assertNotEqual(error.config.path, '');
+                const path = realm.path;
+                realm.close();
+                Realm.Sync.initiateClientReset(path);
+                // open Realm with error.config, and copy required objects a Realm at `path`
```

Comment:
is this a TODO or some information for what is happening ?

Reply:
I remove it - mostly a note about what to do in a real-world app but doesn't make sense in a test